src/pgwire/src/protocol.rs

```diff
@@ -1386,6 +1401,97 @@ where
         Ok(State::Ready)
     }
 
+    async fn read_rows(
```

Comment:
can this have the word `copy` somewhere in the name? `read_rows` is too vague for what it does.

Reply:
right, I think `copy_from` is more appropriate. What do you think?